app/src/main/res/values-it/strings_help.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Help -->
+    <string name="help_navigation_title">Navigazione</string>
+    <string name="help_navigation">
+        Trascina in alto la schermata principale per mostrare tutte le applicazioni, e in basso per tornare indietro.
+        \n
+        \nTrascina in basso la schermata principale per mostrare i preferiti, e in alto per tornare indietro.
+        \n
+        \nClicca sul punsante menu (quando sono visualizzati i preferiti) o l\'icona del lanciatore per aprire il menu.
+        \n
+        \nTieni premuta un\'applicazione o una cartella per accedere alle sue impostazioni.
+    </string>
+    <string name="help_notification_title">Notifica</string>
+    <string name="help_notification">
+        Mostra un pop-up dei preferiti al di sopra delle altre applicazioni quando lasci la schermata principale.
+    </string>
+    <string name="help_list_update_title">Aggiornamento della lista</string>
+    <string name="help_list_update">
+        Viene visualizzato un messaggio pop-up quando la lista delle applicazioni e le icone vengono aggiornate,
+        quando le applicazioni vengono avviate, aggiunte o rimosse, oppure se il pacchetto icone viene modificato.
+    </string>
+    <string name="help_folders_title">Cartelle</string>
+    <string name="help_folders">
+        Gestisci le cartelle o aggiungile ai preferiti dal pulsante menu, oppure clicca il titolo di una cartella aperta
+        per creare, rinominare, e inserirci applicazioni.
+        \nRimuovere una cartella ricolloca tutte le sue applicazioni nella lista principale.
+    </string>
+    <string name="help_search_title">Cerca</string>
+    <string name="help_search">
+        Trova qualsiasi applicazione tramite l\'icona cerca. Può anche essere inserita tra i preferiti in modo da
+        essere disponibile dalla notifica, oppure nascosta.
+        \n
+        \nPremere \"Invio\" apre il primo risultato di ricerca.
+    </string>
+
+    <!-- Changelog (the string "changelog_folder" must be "changelog-en" or "changelog-fr") -->
+    <string name="changelog_mentions">\"@pseudo\" indica l\'utente che ha suggerito l\'idea o che ha segnalato il bug</string>
+    <string name="changelog_folder">changelog-it</string>
```

Comment:
This should be either "changelog-en" or "changelog-fr" (as feels most appropriate) as only those exist.

It is used to display the history of release notes published in F-Droid which I write only in English and French. Since it is very temporary information (only the time of a release) and most written during publishing process (at the last moment), it doesn't worth the effort and struggle to have them translated in other languages.

Reply:
It's reasonable, you are right, I missed it :). Updated.